app/Eco/Contact/Contact.php

```diff
@@ -67,6 +68,11 @@ public function addresses()
         return $this->hasMany(Address::class);
     }
 
+    public function addressesActive()
+    {
+        return $this->hasMany(Address::class)->where('type_id', '!=', 'old')->orWhere('end_date', '>=', Carbon::parse('now')->format('Y-m-d'));
```

Comment:
In plaats van `->hasMany(Address::class)->` kun je ook `->addresses()->`

Reply:
Changed in code.